cxx-qt/src/lib.rs

```diff
@@ -118,13 +116,11 @@ pub fn qobject(_args: TokenStream, _input: TokenStream) -> TokenStream {
 /// #[cxx_qt::bridge]
 /// mod my_object {
 ///     #[derive(Default)]
-///     struct Data {
+///     struct RustObj {
```

Comment:
This is missing a `#[cxx_qt::qobject]` attribute

Reply:
Done